sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/quickpulse/model/OpenTelMetric.java

```diff
@@ -0,0 +1,52 @@
+package com.azure.monitor.opentelemetry.exporter.implementation.quickpulse.model;
+
+import java.util.ArrayList;
+import java.time.LocalDateTime;
+
+public class OpenTelMetric {
+
+    private String name;
+    private ArrayList<OpenTelDataPoint> dataPoints;
+    private LocalDateTime lastTimestamp;
+
+    public OpenTelMetric(String name) {
+        this.name = name;
+        this.dataPoints = new ArrayList<>();
+        this.lastTimestamp = LocalDateTime.now();
+    }
+
+    public void addDataPoint(double value) {
+        OpenTelDataPoint dataPoint = new OpenTelDataPoint(value);
+        this.dataPoints.add(dataPoint);
+        this.updateLastTimestamp();
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public ArrayList<OpenTelDataPoint> getDataPoints() {
+        return dataPoints;
+    }
+
+    public ArrayList<Double> getDataValues() {
+        ArrayList<Double> values = new ArrayList<>();
```

Comment:
See my previous comment & update this accordingly: https://github.com/Azure/azure-sdk-for-java/pull/41220/files#r1688754048

Reply:
refer to https://github.com/Azure/azure-sdk-for-java/pull/41220#discussion_r1690532537